src/openforms/config/service.py

```diff
@@ -0,0 +1,39 @@
+from dataclasses import dataclass
+
+from openforms.contrib.brk.checks import BRKValidatorCheck
+from openforms.forms.models.form import Form
+from openforms.plugins.exceptions import InvalidPluginConfiguration
+
+
+@dataclass
+class BrokenConfiguration:
+    config_name: str
+    exception_message: str | None
+
+
+def check_BRK_config_for_addressNl() -> str | None:
+    live_forms = Form.objects.live()
+
+    for form in live_forms:
+        if form.form_uses_component("addressNL"):
+            try:
+                BRKValidatorCheck.check_config()
+            except InvalidPluginConfiguration as e:
+                return e.args[0]
+            break
+
+    return
```

Comment:
I think something like this might be more readable:

```suggestion
    if any(form.form_uses_component("addressNL") for form in live_forms):
        try:
            BRKValidatorCheck.check_config()
        expect InvalidPluginConfiguration as e:
            return e.args[0]
```

`any` will stop as soon as it encounters a truthy value so performance is the same here

Reply:
On a side note, the way `InvalidPluginConfiguration` is being defined makes it unsafe to use `e.args[0]`, probably at some point we could define a proper `__init__` for this exception class

Reply:
Why is it unsafe?Could you please elaborate?

Reply:
It is defined as a simple subclass:
https://github.com/open-formulieren/open-forms/blob/fac58af05a9685d68068ba1f4609c748079d96ac/src/openforms/plugins/exceptions.py#L1-L2

Meaning I can do `e = InvalidPluginConfiguration()` and `e.args == tuple()`